kitten-app/src/services/createCatServices.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+import type { UnsavedCatname, APIResult } from "../types";
+
+const createCatServices = async (unsavedcat: UnsavedCatname): Promise<APIResult<string>> => {
```

Comment:
[question] Does this method create a cat-service? 
What other methods would `createCatService` contain and what would that mean for reusability?
What does will the result of the API be and would it be possible to be more explicit, and if so, how?